arex-storage-model/src/main/java/com/arextest/model/replay/PagedRequestType.java

```diff
@@ -24,4 +27,5 @@ public class PagedRequestType {
   private String sourceProvider;
   private List<SortingOption> sortingOptions;
   private Map<String, String> tags;
+  private List<String> excludeFields = Arrays.asList(TARGET_RESPONSE_COLUMN_NAME, EIGEN_MAP_COLUMN_NAME);
```

Comment:
Suggest to use `String[] includeExtendFields` for compatibility with the current request